es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = number + street;
+
+    console.log(`LearningFuze ${address}`);
```

Comment:
This was _not_ a concatenation, it was a method call with two arguments:

---

![image](https://user-images.githubusercontent.com/16143771/156110304-aee1b71d-477d-446b-abdd-f846e4b81518.png)

---

![image](https://user-images.githubusercontent.com/16143771/156110355-85deb028-4fb5-49f4-a66e-407f0c60464e.png)